quipucords/scanner/vcenter/inspect.py

```diff
@@ -151,10 +150,13 @@ def parse_host_props(self, props, cluster_dict):
         facts = {}
         for prop in props:
             if prop.name == 'parent':
-                facts['host.cluster'] = \
-                    cluster_dict[str(prop.val)]['cluster.name']
-                facts['host.datacenter'] = \
-                    cluster_dict[str(prop.val)]['cluster.datacenter']
+                c_src = cluster_dict.get(str(prop.val))
+                if c_src is not None:
+                    facts['host.cluster'] = c_src.get('cluster.name')
+                    facts['host.datacenter'] = c_src.get('cluster.datacenter')
+                else:
+                    facts['host.cluster'] = None
```

Comment:
I think we can leave these out

Reply:
@elyezer left a comment that really improved this section of code.